src/module.c

```diff
@@ -156,12 +156,12 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 		return REDISMODULE_ERR;
 	}
 
-	if(RedisModule_CreateCommand(ctx, "graph.EXPLAIN", CommandDispatch, "write", 1, 1,
+	if(RedisModule_CreateCommand(ctx, "graph.EXPLAIN", CommandDispatch, "readonly", 1, 1,
```

Comment:
explain is a write command, it can create a missing graph.

Reply:
so it should be OOM too